WordPress/Tests/Theme/NoTitleTagUnitTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Unit test class for WordPress Coding Standard.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     https://make.wordpress.org/core/handbook/best-practices/coding-standards/
+ */
+
+/**
+ * Unit test class for the NoTitleTag sniff.
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @author    carolinan
+ */
+class WordPress_Tests_Theme_NoTitleTagUnitTest extends AbstractSniffUnitTest {
+
+	/**
+	 * Returns the lines where errors should occur.
+	 *
+	 * The key of the array should represent the line number and the value
+	 * should represent the number of errors that should occur on that line.
+	 *
+	 * @return array(int => int)
+	 */
+	public function getErrorList() {
+		return array(
+			7 => 1,
+			9 => 1,
+			19 => 1,
+			24 => 1,
+			26 => 1,
+			28 => 1,
+			// PHP 5.2 has an issue tokenizing `<s` so splits the string into two.
+			30 => version_compare( PHP_VERSION, '5.3.0', '>=' ) ? 1 : 2,
```

Comment:
The line 30 `<title>my title</title> <svg width="500" height="300" >...</svg> <title>my title</title> <!-- Bad. -->`

gets tokenized like this

```
string(36) "<title>my title</title> "
string(5) "<s"
string(124) "vg width="500" height="300" >...</svg> <title>my title</title> <!-- Bad. -->
"
```
whereas in higher PHP versions the token would be
`string(165) "<title>my title</title> <svg width="500" height="300" >...</svg> <title>my title</title> <!-- Bad. -->
"`